google/cloud/gkeconnect/CMakeLists.txt

```diff
@@ -26,8 +26,10 @@ if (BUILD_TESTING AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
     google_cloud_cpp_add_common_options(gkeconnect_quickstart)
     add_test(
         NAME gkeconnect_quickstart
-        COMMAND cmake -P "${PROJECT_SOURCE_DIR}/cmake/quickstart-runner.cmake"
-                $<TARGET_FILE:gkeconnect_quickstart> GOOGLE_CLOUD_PROJECT)
+        COMMAND
+            cmake -P "${PROJECT_SOURCE_DIR}/cmake/quickstart-runner.cmake"
+            $<TARGET_FILE:gkeconnect_quickstart> GOOGLE_CLOUD_PROJECT
+            GOOGLE_CLOUD_CPP_TEST_REGION "fakedmembership")
```

Comment:
The CMake helper function is actually accepting variables as input. So this thing will look up the variable called "fakedmembership".

Feel free to use any other variable, maybe with a comment saying that we don't expect it to be valid anyway.

Reply:
done